src/components/navigationBar/NavigationBar.tsx

```diff
@@ -0,0 +1,63 @@
+import * as React from "react";
+import theme from "../../theme";
+import { SVGIcon } from "../svgicon";
+import { BurgerWrapper, NavigationBarWrapper } from "./styles";
+import { Props } from "./types";
+import { useMenuBarConfiguration } from "./utils/hooks";
+import { NavigationBarProvider } from "./utils/provider";
+
+export function NavigationBar(props: Props) {
+  return (
+    <NavigationBarProvider popoverConfig={props.popoverConfig}>
+      <NavigationBarVisual {...props} />
+    </NavigationBarProvider>
+  );
+}
+
+function NavigationBarVisual({
+  position = "fixed",
+  expandable = true,
+  height = "100vh",
+  backgroundColor = theme.color.background.ui01,
+  closeOnLeave = false,
+  expandableConfig = {
+    timingFunction: "linear",
+    duration: "0.2s",
+  },
+  children,
+}: Props) {
+  const { expanded, expandedDone, handleExpand, setExpanded, setExpandedDone } =
+    useMenuBarConfiguration();
+
+  if (children == null) {
+    return null;
+  }
```

Comment:
Maybe we could add console.warn here to notify the user that he passed `null` and component won't work?